readthedocs/embed/tests/test_api.py

```diff
@@ -36,6 +36,9 @@ def setup_method(self, settings):
         settings.USE_SUBDOMAIN = True
         settings.PUBLIC_DOMAIN = 'readthedocs.io'
 
+    def get(self, client, *args, **kwargs):
+        return client.get(*args, **kwargs)
```

Comment:
Probably could use a docstring explaining why this is here -- I assume to override elsewhere?

Reply:
I just added one. It's overridden in the proxied api tests (in this same file)